test/boost/memtable_test.cc

```diff
@@ -1271,6 +1271,8 @@ static future<> exceptions_in_flush_helper(std::unique_ptr<sstables::file_io_ext
 
         co_await env.execute_cql(fmt::format("create table t0 (pk text primary key, v text)"));
 
+        schema_filter = env.local_db().find_column_family("ks", "t0").schema().get();
```

Comment:
I'm not sure I understand why it's safe to keep around the raw (non-smart) pointer to the schema - which theoretically can change over time. Wouldn't it be safer to just call this find_column_family every time?

But since this is only a test, and I guess it does work (you tested) and nothing can cause the schema to change in the middle of the test, then this is ok too.